lib/rules/value-no-vendor-prefix/__tests__/index.mjs

```diff
@@ -175,7 +172,7 @@ testRule({
 
 	accept: [
 		{
-			code: 'a { $margin: -webkit-box; }',
+			code: 'a { $foo: -webkit-plaintext; }',
```

Comment:
This was setup so that it could pick regressions, hence the value needs to be one of the unprefixable ones.